### PR TITLE
defect- 13091433

### PR DESCRIPTION
**Pull request by CHenrie**
@jbest84 Please Review

### CODE REVIEW

File: src/Views/History/Edit.js [hunk 1]
```diff
@@ -354,8 +355,6 @@ define('Mobile/SalesLogix/Views/History/Edit', [
                 };
                 lookup[this.context.resourceKind].call(this, this.context);
             }
-
-            this.fields['Text'].setValue(values['LongNotes'] || values['Notes'] || '');
```
Comment (jbest84) [inline src/Views/History/Edit.js:358]: @CHenrie Indicated this line was useless due to default values getting set after.

**Comment by jbest84** [comment]
@CHenrie Accept.

**Comment by slxbuilder** [comment]
Merged build triggered.

**Comment by slxbuilder** [comment]
Merged build started.

**Comment by slxbuilder** [comment]
Merged build finished.

**Comment by jbest84** [comment]
Defect fixed in DTS. Merging.

**Pull request merged**